### ISSUE TITLE
How to modularize trackers with loopback?

### ISSUE THREAD

**Issue by tsuberim** [daostack/daostack-node#5]
Ideally we want each tracker to live in its own folder with a subfolder `models`. I suggest writing a CLI that can generate trackers according to a template and register them automatically with loopback. (already have a very crude version of this which just creates a tracker from a template)

**Comment by tibetsprague** [comment]
yes i think that could work. maybe the migrate script can automatically look for all models in the trackers directory then just model-config.json would have to be updated by the CLI

**Comment by tsuberim** [comment]
Could also be nice to suggest automatically generated models based on the ABI of a contract
